crypto/bio/b_addr.c

```diff
@@ -565,7 +566,8 @@ static int addrinfo_wrap(int family, int socktype,
                          unsigned short port,
                          BIO_ADDRINFO **bai)
 {
-    OPENSSL_assert(bai != NULL);
+    if (bai == NULL)
```

Comment:
Well, as this was previously an OPENSSL_assert(), if anything ever did rely on it then it would have crashed.